change_log/next/FE-1924-adds-deprecated-option-to-headings.yml

```diff
@@ -0,0 +1,4 @@
+New Features: Adds `utils/helpers/warn-as-deprecated` function to warn of deprecated components
+Other: |
+  Updates `.renogen` headings to include a `Deprecated Components` option
+  Creates a `__deprecated__` directory and moves and`Textbox` into it and updates paths
```

Comment:
```suggestion
  Creates a `__deprecated__` directory and moves and `Textbox` into it and updates paths
```

Reply:
Maybe also need a line here that `Textbox` was deprecated - so if you are using that component you need to update your imports from `components/textbox` to `__depcated__/components/textbox`